docs/getting-started/quickstart.md

```diff
@@ -7,7 +7,9 @@ import TabItem from '@theme/TabItem';
 
 # DBOS Quickstart
 
-This guide shows you how to deploy your first app to DBOS Cloud in less than 5 minutes. After that, it shows you how to develop DBOS apps locally.
+You start developing a DBOS app by creating a folder for it on your computer. From this folder, you can run the app locally and deploy to DBOS Cloud in minutes.
+![DBOS Development and Deployment Model](./assets/quickstart-diagram.png)
```

Comment:
The image looks great in light mode but terrible in dark mode. How hard is it to create a dark mode version of it?

Reply:
will do

Reply:
Done